core/network/fabric-interop-cc/contracts/interop/manage_assets.go

```diff
@@ -101,7 +145,7 @@ func (s *SmartContract) LockAsset(ctx contractapi.TransactionContextInterface, a
 		return "", errors.New(errorMsg)
 	}
 
-	err = ctx.GetStub().PutState(string(contractId), assetLockKeyBytes)
+	err = ctx.GetStub().PutState(contractIdPrefix + string(contractId), assetLockKeyBytes)
```

Comment:
If you are not using the `contractId` field in this function for any other purpose, perhaps you can return the string with the prefix from the function that generates the contract ID itself?

Reply:
@VRamakrishna , the prefix here is internal to `interop-cc` and helps avoid conflicts with other types of keys on the ledger. Hence, I prefer not to expose to the user.

Reply:
@VRamakrishna crated a function `generateContractIdMapKey` to return the key for the map using contractId.